cactusbot/api.py

```diff
@@ -10,6 +10,17 @@ class CactusAPI(API):
 
     URL = "https://cactus.exoz.one/api/v1/"
 
+    SCOPES = {
```

Comment:
@Innectic No - a set is perfectly fine. There isn't an order to these, after all.